templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width">
+    <link rel="SHORTCUT ICON" href="{{ FaviconUrl }}" />
+    {{{ open_sans_stylesheet }}}
+    {{{ ng_checkout_stylesheet }}}
```

Comment:
I think we should try to inject the default UCO stylesheet as part of `{{{ checkout_content }}}` instead of having to include separate Handlebars objects `{{{ ng_checkout_stylesheet }}}` and `{{{ open_sans_stylesheet }}}`. In other words, in `angularcheckout/checkout_content.html` (BCApp), write a script to inject both stylesheets to the top of `<head>`